ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-master.yaml

```diff
@@ -86,6 +86,25 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: hypershift-aws-e2e-cluster
+- as: e2e-pao-updating-profile-hypershift
+  always_run: false
```

Comment:
Hey @jmencak I recall you prefer to have it this way and not always running.

Just wanted to make sure you're pleased with that

Reply:
Hey Talor, cannot remember the conversation, but yes, unless we need the tests blocking merges, this makes sense.  Do we also want `optional: true` here?

Reply:
@jmencak not necessarily. 
If this lane intended to run only if mentioned explicitly, then the least I would expect is for it to be required and not optional.